github/Team.py

```diff
@@ -447,6 +487,8 @@ def _useAttributes(self, attributes):
             self._name = self._makeStringAttribute(attributes["name"])
         if "description" in attributes:  # pragma no branch
             self._description = self._makeStringAttribute(attributes["description"])
+        if "organization" in attributes and "url" in attributes["organization"]:  # pragma no branch
+            self._organization_url = self._makeStringAttribute(attributes["organization"]["url"])
```

Comment:
Oh, is it an actual organization? We should just set that then --

```
self._organization = self._makeClassAttribute(github.Organization.Organization, attributes["organization"])

Reply:
We already set _organization in fact! Why are you adding a _url parameter?

Reply:
Oh. Indeed. Thanks for pointing this out. Fixed in https://github.com/PyGithub/PyGithub/pull/1509/commits/b7bea98e05a4fdc7d1de46d2d2e3b1697a0ea7e5